src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +49,11 @@ public Movie add(Movie movie) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.createQuery("FROM Movie ",
+                    Movie.class).getResultList();
+        } catch (Exception e) {
+            throw new DataProcessingException("Can't get all movies", e);
```

Comment:
Lets use uniform exception messages (in CinemaHallDaoImpl class getAll() method you wrote "An error occurred while processing query to get all cinema halls")

Reply:
done